examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -0,0 +1,135 @@
+# Basic Python CI With Cloud Source Repositories and Cloud Build
+
+## Overview
+This repo contains code and instructions for using Google Cloud Source Repositories and Cloud Build to execute unit tests upon code check-in.
+
+The basic steps we need to follow are:
+1. Create a new project in Cloud Source Repository and clone it to your machine
```

Comment:
Repository=Repositories.

Reply:
That is a great catch.  I have no idea how you saw that one, thank you!